src/VolleyManagement.Backend/VolleyManagement.Services/Authorization/UserService.cs

```diff
@@ -21,12 +21,13 @@ public class UserService : IUserService
         private readonly IAuthorizationService _authService;
         private readonly IQuery<User, FindByIdCriteria> _getUserByIdQuery;
         private readonly IUserRepository _userRepository;
-        private readonly IQuery<List<User>, GetAllCriteria> _getAllUsersQuery;
+        private readonly IQuery<ICollection<User>, GetAllCriteria> _getAllUsersQuery;
         private readonly IQuery<Player, FindByIdCriteria> _getUserPlayerQuery;
         private readonly ICacheProvider _cacheProvider;
-        private readonly IQuery<List<User>, UniqueUserCriteria> _getAdminsListQuery;
+        private readonly IQuery<ICollection<User>, UniqueUserCriteria> _getAdminsListQuery;
         private readonly ICurrentUserService _currentUserService;
 
+#pragma warning disable S107 // Methods should not have too many parameters
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)